releasenotes/notes/ndm-collect-vpn-tunnels-metrics-and-options-7685e040e9580dc1.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Adds Cisco IPsec VPN tunnels metrics and options data collection to SNMP. This can be enabled/disabled using the `collect_vpn` config.
```

Comment:
```suggestion
    Adds Cisco IPsec VPN tunnel metrics and options data collection to SNMP. This can be enabled and disabled using the `collect_vpn` config.
```